.cbmc-batch/stubs/s_sift_either_override.c

```diff
@@ -0,0 +1,10 @@
+
+#include <aws/common/priority_queue.h>
+
+bool __CPROVER_file_local_priority_queue_c_s_sift_either(struct aws_priority_queue *queue, size_t index) {
```

Comment:
I comment would be helpful here for documentation. Why do we need this stub?

Reply:
I agree with @feliperodri, the current comment is not sufficient. I would like to see two things please:
1. Why do we need the stub?
1. Why is it safe to stub the original function out?

Reply:
You asked for a re-review but there's still no comment here, please add one.

Reply:
I think I added comments in the stub. This is the [link](https://github.com/awslabs/aws-c-common/pull/398/files#diff-2d8a8fc92c4ceab7fa583e22798ca1ff) with the file changes.